cypress/fixtures/commonComponents/dateInput.json

```diff
@@ -33,13 +33,11 @@
   "large": {
     "mt": "400"
   },
-  "default": {
-    
-  },
+  "default": {},
   "minDate": {
-    "minDate": "2019-04-04"
+    "minDate": "2030-04-04"
```

Comment:
it will be overridden by the `const today = Cypress.dayjs().format("YYYY-MM-DD");` in helper file

Reply:
No it won't because I removed that 🙂